### PR TITLE
cli-registries-update

### PR DESCRIPTION
**Pull request by resmaa**
**Release 3.0 
cli and registries update**

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/932#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @resmaa. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: content/docs/16-component.md
Comment (truegilb) [inline content/docs/16-component.md:24]: should be 3.0.0

File: content/docs/16-component.md
Comment (truegilb) [inline content/docs/16-component.md:25]: should be 3.0.0

File: content/docs/16-component.md
Comment (truegilb) [inline content/docs/16-component.md:43]: should be 3.0.0

File: content/docs/16-component.md
Comment (truegilb) [inline content/docs/16-component.md:55]: should be 3.0.0

File: content/docs/16-component.md
Comment (truegilb) [inline content/docs/16-component.md:77]: should be 3.0.0

File: content/docs/16-component.md
Comment (truegilb) [inline content/docs/16-component.md:89]: should be 3.0.0

File: content/docs/16-component.md
Comment (truegilb) [inline content/docs/16-component.md:101]: should be 3.0.0

**Review by truegilb — CHANGES_REQUESTED** [review]
Many places still reference 2.9.0

**Review by truegilb — APPROVED** [review]
Looks good.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/932#" title="Author self-approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/932#pullrequestreview-1152538018" title="Approved">truegilb</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
